papis/commands/doctor.py

```diff
@@ -299,14 +311,28 @@ def fixer_author_list_from_author() -> None:
         else:
             return None
 
-    return [Error(name=KEYS_MISSING_CHECK_NAME,
-                  path=folder,
-                  msg=f"Key '{k}' does not exist",
-                  suggestion_cmd=f"papis edit --doc-folder {folder}",
-                  fix_action=make_fixer(k),
-                  payload=k,
-                  doc=doc)
-            for k in keys if k not in doc]
+    errors = []
+    for key in keys:
+        if key not in doc:
+            errors.append(Error(name=KEYS_MISSING_CHECK_NAME,
+                                path=folder,
+                                msg=f"Key '{key}' does not exist",
+                                suggestion_cmd=f"papis edit --doc-folder {folder}",
+                                fix_action=make_fixer(key),
+                                payload=key,
+                                doc=doc))
+
+        elif not _is_nonempty(doc[key]):
+            errors.append(Error(name=KEYS_MISSING_CHECK_NAME,
+                                path=folder,
+                                msg=f"Value contains one or more empty entries:\n{
+                                papis.document.dump({f"{key}":doc[key]})
+                                }",
+                                suggestion_cmd=f"papis edit --doc-folder {folder}",
+                                fix_action=None,
+                                payload=key,
+                                doc=doc))
```

Comment:
I'm not sure this conceptually fits in here very well, since this check is about "keys" missing, not empty values?

Maybe worth adding very simple copy-pasted `values-missing` check instead? I think I argued a bit for adding it to the `key-type` check, since an "empty value" might be an "incorrect value type", but it just complicates the logic.